src/main/java/de/berlin/lostberlin/service/MailService.java

```diff
@@ -0,0 +1,56 @@
+package de.berlin.lostberlin.service;
+
+import com.sendgrid.*;
+import de.berlin.lostberlin.model.Business;
+import de.berlin.lostberlin.model.Order;
+import de.berlin.lostberlin.repository.BusinessRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.io.IOException;
+import java.sql.SQLException;
+import java.util.Optional;
+
+public class MailService {
+    @Autowired
+    private static BusinessRepository businessRepo;
+
+    public static Response sendEmail(Order order, MailTypes mailType) throws IOException, SQLException {
+        Email from = new Email("aoneko@gmx.com");
+        String subject = "";
+        Email to = new Email(order.getEmail());
+        Content content = null;
+        if (mailType == MailTypes.CONFIRMATION) {
+            subject = "Order confirmation";
+            content = new Content("text/html", "<h1>Hi, " + order.getName() + "</h1 > \n" +
```

Comment:
I would create more methods here. For example: sendConfirmationMail() and sendNotificationMail() and they are different in only the text template, which can be externalised to some property files, which placeholders are populated from some object, for instance, a HashMap. 

Then you can get rid of the if for mail type and the method becomes more readable if email text is externalised.

Reply:
thanks, it makes sense. will change it.